zerver/decorator.py

```diff
@@ -363,6 +366,8 @@ def authenticate_log_and_execute_json(request, view_func, *args, **kwargs):
         raise JsonableError("Account not active")
     if user_profile.realm.deactivated:
         raise JsonableError("Realm for account has been deactivated")
+    if user_profile.is_incoming_webhook:
+        raise JsonableError("Account is not allowed to log")
```

Comment:
This error message should be more clear to a user -- "Webhook bots can only access webhooks"